package.json

```diff
@@ -9,16 +9,25 @@
   },
   "scripts": {
     "start": "./entry.sh",
-    "build": "coffee -c src",
+    "build": "coffee -c src && webpack --optimize-minimize --define process.env.NODE_ENV=\"production\"",
```

Comment:
They can, but what the optimize-minimize option does is rather complex (they [require two plugins with some arguments](https://github.com/webpack/webpack/blob/57825153e531a079eb0dbd2528e47f4cbafcc7f2/bin/convert-argv.js#L443) so I'd rather leave this like this as it looks way simpler.